Docker/build/WDQS/README.md

```diff
@@ -34,7 +34,9 @@ Variable                 | Default            | Since   | Description
 `WDQS_PORT`              | "9999"             | 0.2.5   | Port of the WDQS host (this service)
 `WDQS_ENTITY_NAMESPACES` | "120,122"          | 0.2.5   | Wikibase Namespaces to load data from
 `WIKIBASE_MAX_DAYS_BACK` | "90"               | 0.3.0   | Max days updater is allowed back from now
+`BLAZEGRAPH_EXTRA_OPTS`  | ""                 | 0.3.40  | Extra options to be passed to Blazegraph
```

Comment:
```suggestion
`BLAZEGRAPH_EXTRA_OPTS`  | ""                 | 0.3.115  | Extra options to be passed to Blazegraph
```

Reply:
This will be 0.3.115 as in https://github.com/wmde/wikibase-release-pipeline/pull/357

Reply:
Infact, that might not even make any sense, I think we would need to put wmde.9 here how as the versioning of the images and how this repo builds things is different

Reply:
I was about to ask how I would have known what version to use. So I'll use `wmde.9` ?

Reply:
Yup, wmde.9 will be the next release!